app/src/main/java/com/soen490chrysalis/papilio/view/UserProfileActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.soen490chrysalis.papilio.view
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.soen490chrysalis.papilio.databinding.ActivityUserProfileBinding
+
+
+class UserProfileActivity : AppCompatActivity()
+{
+    private lateinit var binding : ActivityUserProfileBinding
+
+    override fun onCreate(savedInstanceState: Bundle?)
+    {
+        supportActionBar?.hide()
```

Comment:
Why hide the action bar? I want it to be present and you should add a back button to it

Reply:
Done and done. The action bar now displays the title of the page and features a back button that takes you back to the activity page